inference-engine/src/mkldnn_plugin/nodes/subgraph.cpp

```diff
@@ -0,0 +1,861 @@
+                }
+
+                return std::make_shared<CpuBlockedMemoryDesc>(prc, shape, blocks, order, offset);
+                // TODO: need investigate
+                // bad accuracy for shape {1, 1, 4, 11}, {2, 5, 1, 1}
+                // same for disabled collapse dims
+            } else if (lt == Blocked && shape.getRank() != 1 && (shape.getMinDims()[1] != Shape::UNDEFINED_DIM && shape.getMinDims()[1] > 1)) {
+                size_t blockSize = mayiuse(dnnl::impl::cpu::x64::avx512_common) ? 16 : 8;
+
+                VectorDims blocks = dims;
+                VectorDims order(blocks.size());
+                std::iota(order.begin(), order.end(), 0);
+
+                blocks[1] = dims[1] != Shape::UNDEFINED_DIM ? div_up(blocks[1], blockSize) : Shape::UNDEFINED_DIM;
+                blocks.push_back(blockSize);
+                order.push_back(1);
+
+                return std::make_shared<CpuBlockedMemoryDesc>(prc, shape, blocks, order, offset);
+            } else {
+                VectorDims blocks = dims;
+                VectorDims order(blocks.size());
+                std::iota(order.begin(), order.end(), 0);
+
+                return std::make_shared<CpuBlockedMemoryDesc>(prc, shape, blocks, order, offset);
+            }
+        };
+
+        // Todo: inhereted from eltwise node. Why offset is max for non-dynamic nodes?
+        //  size_t offset = isDynamicNode() ? 0 : std::numeric_limits<size_t>::max();
+        size_t offset = std::numeric_limits<size_t>::max();
+        NodeConfig config;
+        config.dynBatchSupport = outputShapes[0].getRank() > 1 && inputShapes[0] == outputShapes[0];
+//        config.inConfs.resize(getParentEdges().size());
+//        for (size_t i = 0; i < getParentEdges().size(); i++) {
+        config.inConfs.resize(inputShapes.size());
+        for (size_t i = 0; i < inputShapes.size(); i++) {
+            PortConfig portConfig;
+            portConfig.inPlace = (!i && canBeInPlace() && inputPrecisions[i] == supportedPrecision) ? 0 : -1;
+            portConfig.constant = false;
+            portConfig.desc = createMemoryDesc(inputShapes[i], supportedPrecision, offset);
+            config.inConfs[i] = portConfig;
+        }
+        config.outConfs.resize(outputShapes.size());
+        for (size_t i = 0; i < outputShapes.size(); i++) {
+            PortConfig portConfig;
+            portConfig.inPlace = -1;
+            portConfig.constant = false;
+            portConfig.desc = createMemoryDesc(outputShapes[i], supportedPrecision, offset);
+            config.outConfs[i] = portConfig;
+        }
+
+        impl_desc_type impl_type;
+        if (mayiuse(x64::avx512_common)) {
+            impl_type = impl_desc_type::jit_avx512;
+        } else if (mayiuse(x64::avx2)) {
+            impl_type = impl_desc_type::jit_avx2;
+        } else {
+            // Should never hit here: currently only avx512 and avx2 are supported
+            impl_type = impl_desc_type::unknown;
```

Comment:
I believe there's no need in this, since we check the isa during tokenization stage. I just wanted to provide a default value here. It is probably better to do it right after the impl_type declaration, so I did.